src/arch/x86/lib/cpu_performance/cpu_info.h

```diff
@@ -1,13 +1,15 @@
 #ifndef X86_CPU_INFO_H_
 #define X86_CPU_INFO_H_
 
+struct cpu_feature {
```

Comment:
As this structure is arch-agnostic, it's better to put it in `src/include/hal/cpu_info.h`

Reply:
Done in the latest fixup commit!